src/linting_error/mod.rs

```diff
@@ -17,4 +17,6 @@ pub enum LintingError {
     NoSpaceAfterColonPrecedingTypeAndScope,
     /// Commit title has no description after the type and scope.
     NoDescriptionAfterTypeAndScope,
+    /// Commit message subject line exceeds the recommended maximum length.
+    MessageTooLong,
```

Comment:
Can you change it to something like CommitTitleTooLong, message isn't very descriptive.